app/views/spree/admin/products/_form.html.haml

```diff
@@ -45,7 +45,7 @@
 
     = f.field_container :shipping_categories do
       = f.label :shipping_category_id, t(:shipping_categories)
-      = f.collection_select(:shipping_category_id, @shipping_categories, :id, :name, { :include_blank => t(:none) }, { :class => 'select2' })
+      = f.collection_select(:shipping_category_id, @shipping_categories, :id, :name, { :class => 'select2' })
```

Comment:
You removed the third parameters (previously was `{ :include_blank => t(:none) }`), so the 4th parameter is considered as the third. 
Anyway, you probably should remplace `{ :include_blank => t(:none) }` by `{ }`, which is the following suggestion:

```suggestion
      = f.collection_select(:shipping_category_id, @shipping_categories, :id, :name, {} , { :class => 'select2' })
```

Reply:
Awesome. Thank you!

Reply:
Do you think there should be a spec for this?
Can there be, with reasonable amount of work?

Reply:
I don't think this small modification deserves a spec. I might be wrong, maybe let's @filipefurtad0 answer this. 

By the way, I think this PR needs commits to be squashed to have a clean git history.